### DISCUSSION TITLE
portable-release-stable

### DISCUSSION THREAD

**Discussion by ehristoforu** [ehristoforu/DeFooocus#4] [Q&A]
### Portable version

**This is a portable release, it will always be relevant.**

![portable-preview](https://github.com/ehristoforu/DeFooocus/assets/136880927/e749954f-3a24-44cc-b103-d4177a271020)

_You need_:
1. download `PortableGit-2.44.0-64-bit.7z.exe` and run it for install git
2. download `DeFooocus_win64_portable.7z` and unzip it
3. run the `install.bat` file
4. run the file `run_{preset}.bat` with the desired preset

<hr /><em>This discussion was created from the release <a href='https://github.com/ehristoforu/DeFooocus/releases/tag/portable-release'>portable-release-stable</a>.</em>

**Comment by departurechen** [comment]
.\python_embeded\python.exe: can't open file 'D:\\DeFooocus_win64_portable_2\\DeFooocus\\entry_with_update.py': [Errno 2] No such file or directory

**Reply by ehristoforu** [reply]
Firstly run ```install.bat```, then run ```run_....bat```

**Reply by babulshrivas** [reply]
> Firstly run `install.bat`, then run `run_....bat`

it still shows the same thing what should i do

**Comment by llueLeo** [comment]
CMD Failed requirements: install -r "requirements_versions.txt"
Loaded preset: C:\AI\DeFooocus_win64\DeFooocus\presets\realistic.json
Total VRAM 8192 MB, total RAM 32581 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce RTX 3060 Ti : native
VAE dtype: torch.bfloat16
Using split optimization for cross attention
Traceback (most recent call last):
  File "C:\AI\DeFooocus_win64\DeFooocus\entry_with_update.py", line 46, in <module>
    from launch import *
Refiner unloaded.
  File "C:\AI\DeFooocus_win64\DeFooocus\launch.py", line 128, in <module>
    from webui import *
  File "C:\AI\DeFooocus_win64\DeFooocus\webui.py", line 16, in <module>
    from modules.rembg import rembg_run
  File "C:\AI\DeFooocus_win64\DeFooocus\modules\rembg.py", line 1, in <module>
    from rembg import remove
ModuleNotFoundError: No module named 'rembg'

**Comment by draz-kira** [comment]
No module named 'rembg'

**Comment by dm-cell** [comment]
ModuleNotFoundError: No module named 'rembg'

**Comment by kahs-Games-channel** [comment]
📝 Русификатор для Fooocus

Скопируйте файл ru.json в папку:

DeFooocus\DeFooocus\language\

Запуск Fooocus с русским интерфейсом:
Откройте батник (run.bat) или создайте новый и добавьте в строку запуска:

.\python_embeded\python.exe -s DeFooocus\entry_with_update.py --attention-split --in-browser --theme dark --language ru

или просто:

.\python_embeded\python.exe -s DeFooocus\entry_with_update.py --language ru

Проверка:

Убедитесь, что файл называется ru.json (не ru.json.json).

Файл должен быть в UTF-8.

Совет:
Если интерфейс остался на английском, перезапустите программу, убедившись, что батник корректно сохранён и путь до ru.json правильный.
 ссылка СКАЧАТЬ ФАИЛ [ru.json](https://github.com/user-attachments/files/22280665/ru.json)
<img width="875" height="547" alt="Русификатор" src="https://github.com/user-attachments/assets/8664ec7f-8a91-4ea9-94c8-a1b4f24c496b" />

**Comment by BlackOdyssey95** [comment]
mine gets stuck once it's fully downloaded & won't appear in my download folder, still say's it's downloading even though has finished
